tests/phpunit/phpunit.xml

```diff
@@ -5,6 +5,7 @@
     <php>
         <const name="BLT_ENV" value="ci"/>
         <const name="BLT_ALIAS" value="self"/>
+        <const name="BLT_MULTISITE_NAME" value="default,g"/>
```

Comment:
I thought it would be better, should this value change, that the test itself wouldn't need to be updated. Also, implementing projects would have an easier time overriding this should they wish to run the tests against their own configuration. 

If you'd rather this be hardcoded, I'm fine with making that change, just let me know.